src/core/templates/Perpendicular.tsx

```diff
@@ -30,7 +33,11 @@ export class Perpendicular {
     return (
       <span>
         {linked(perp, perp1)}
-        {perpendicular}
+        {tooltip(
+          strs.perpendicular,
+          definitions.Perpendicular.keyword,
+          definitions.Perpendicular.definition
+        )}
```

Comment:
Generally I think you can simplify the tooltip method so that `tooltip` only takes 2 parameters, `obj` and `definition`, so, in this case, you can call `tooltip(strs.perpendicular, definitions.Perpendicular)`